debian/rules

```diff
@@ -18,10 +18,11 @@ override_dh_auto_configure:
 		-DNEO_OCL_VERSION_MAJOR=$(NEO_MAJOR) -DNEO_OCL_VERSION_MINOR=$(NEO_MINOR) -DNEO_VERSION_BUILD=$(NEO_BUILD) \
 		-DNEO_VERSION_BUILD=${build_id} -DNEO_VERSION_HOTFIX=${build_hf_id} \
 		-DSUPPORT_DG1=TRUE \
-		-DSUPPORT_XE_HP_CORE=TRUE \
```

Comment:
can you explain the rationale behind this flag removal ?

Reply:
according to compute-runtime team, this flag has been deprecated....

Reply:
thanks